core/src/commonMain/kotlin/com/xebia/functional/xef/auto/AI.kt

```diff
@@ -6,7 +6,10 @@ import arrow.core.right
 import com.xebia.functional.xef.AIError
 import com.xebia.functional.xef.embeddings.OpenAIEmbeddings
 import com.xebia.functional.xef.env.OpenAIConfig
-import com.xebia.functional.xef.llm.openai.*
+import com.xebia.functional.xef.llm.openai.AIClient
```

Comment:
`AIClient` should be located in a different package because it's unrelated to `openai`, isn't it?

Reply:
yes, when we move Open ai out of core and depend on https://github.com/aallam/openai-kotlin we will remove the openai package